android/assets/jsons/translations/Portuguese.properties

```diff
@@ -722,97 +722,97 @@ You have won a scientific victory! = Você ganhou uma vitória ciêntifica!
+You have achieved victory through the awesome power of your Culture. Your civilization's greatness - the magnificence of its monuments and the power of its artists - have astounded the world! Poets will honor you as long as beauty brings gladness to a weary heart. = Você alcançou vitória pelo incrível poder de sua Cultura. A grandiosidade de sua civilização - a magnificiência de seus monumentos e o poder de seus artistas - deixou o mundo pasmo! Poetas irão lhe honrar enquanto beleza trouxer alegria a um coração inquieto.
  # Requires translation!
-The world has been convulsed by war. Many great and powerful civilizations have fallen, but you have survived - and emerged victorious! The world will long remember your glorious triumph! = 
+The world has been convulsed by war. Many great and powerful civilizations have fallen, but you have survived - and emerged victorious! The world will long remember your glorious triumph! = O mundo tem estado convulsionado por guerra. Muitas grandes e poderosas civilizações cairam, mas você sobreviveu - e emergiu vitorioso! O mundo recordará por muito tempo seu triunfo glorioso!
  # Requires translation!
-You have achieved victory through mastery of Science! You have conquered the mysteries of nature and led your people on a voyage to a brave new world! Your triumph will be remembered as long as the stars burn in the night sky! = 
+You have achieved victory through mastery of Science! You have conquered the mysteries of nature and led your people on a voyage to a brave new world! Your triumph will be remembered as long as the stars burn in the night sky! = Você alcançou vitória através do domínio da Ciência! Você conquistou os mistérios da natureza e levou seu povo à uma viagem a um admiravel mundo novo! Seu trinfo será recordado enquanto as estrelas brilharem no céu noturno!
  # Requires translation!
-You have been defeated. Your civilization has been overwhelmed by its many foes. But your people do not despair, for they know that one day you shall return - and lead them forward to victory! = 
+You have been defeated. Your civilization has been overwhelmed by its many foes. But your people do not despair, for they know that one day you shall return - and lead them forward to victory! = Você foi derrotado. Sua civilização foi sobrepujada por seus vários inimigos. Mas seu povo não se desespera, pois sabem que um dia você há de retornar - e liderá-los em frente, à vitória!
 One more turn...! = Um turno mais...!
 Built Apollo Program = Completou o programa Apollo
 Destroy [civName] = Destruir [civName]
  # Requires translation!
-Our status = 
+Our status = Nosso status
  # Requires translation!
-Global status = 
+Global status = Status global
  # Requires translation!
-Rankings = 
+Rankings = Posicionamentos
  # Requires translation!
-Spaceship parts remaining = 
+Spaceship parts remaining = Partes de naves espaciais faltando
```

Comment:
Is naves plural? There's just one Spaceship per nation in Civ... Eu ja nao falo Portuges mais.. 'Partes faltando de nostra nave espacial' sounds better to my long-forgotten portuñol

Reply:
Almost right, 'nostra' is not portuguese.
Your suggestions could be: ' Partes faltando de nossa nave espacial'